src/platform/Kconfig

```diff
@@ -204,7 +211,7 @@ endchoice
 
 config MAX_CORE_COUNT
 	int
-	default 4 if TIGERLAKE
+	default 4 if TIGERLAKE || LUNARLAKE
```

Comment:
that's not correct, please check the numbers of cores in the HAS.

Reply:
There are 5 cores in LNL, however Zephyr do not support more than 4 yet. 
Zephyr PR with fix WIP.

Reply:
that's worthy of a comment...

Reply:
Ack, please add a comment and ideally a link to a public PR or bug for the issue.

I understood the limitation to 4 cores only affects spinlock debugging, so this is news to be that we are limited.